src/views/splash/splash.jsx

```diff
@@ -106,12 +114,33 @@ var Splash = injectIntl(React.createClass({
             if (!err) return this.setState({featuredGlobal: body});
         }.bind(this));
     },
-    getFeaturedCustom: function () {
+    getSharedByFollowing: function () {
         api({
-            uri: '/proxy/users/' + this.props.session.session.user.id + '/featured'
+            uri: '/projects/following/users',
+            authentication: this.props.session.session.user.token
```

Comment:
Oh also I forgot to mention @liam4 – the value is not going to be seen here. This is part of an infrastructure change on the backend to how we handle these rows that we're doing to increase their efficiency, and make sure they're not taking up a lot of resources to be retrieved.